test/dlc_tests/sanity/test_security_check.py

```diff
@@ -58,7 +62,11 @@ def is_image_covered_by_allowlist_feature(image):
     image_framework, image_version = get_framework_and_version_from_tag(image)
     if image_framework not in LOWER_THRESHOLD_IMAGES or any(substring in image for substring in ["example"]):
         return False
-    if Version(image_version) >= Version(LOWER_THRESHOLD_IMAGES[image_framework]):
+    if Version(image_version) >= Version(LOWER_THRESHOLD_IMAGES[image_framework]["minimum_version_included"]):
+        if "maximum_version_excluded" in LOWER_THRESHOLD_IMAGES[image_framework]:
+            if Version(image_version) < Version(LOWER_THRESHOLD_IMAGES[image_framework]["maximum_version_excluded"]):
+                return True
+            return False
```

Comment:
Please use `packaging.specifiers.SpecifierSet` to define upper and lower limits.

Reply:
Thanks!! Made the change.